.github/ISSUE_TEMPLATE/1-bug.md

```diff
@@ -0,0 +1,64 @@
+---
+name: 🐛 Bug report
+about: Create a report to help us improve
+---
+
+---
+
+<!--
+  Please check the needed checkboxes ([ ] -> [x]) and fill out the TODOs.
+  Leave the comments as they are: they won’t show on GitHub.
+
+  Some general tips:
+  - Is this really a problem?
+  - Is this a problem here?
+  - Can this be solved in a different way?
+-->
+
+### Initial checklist
+
+*   [ ] I read the support docs <!-- https://github.com/micromark/.github/blob/main/support.md -->
+*   [ ] I read the contributing guide <!-- https://github.com/micromark/.github/blob/main/contributing.md -->
+*   [ ] I agree to follow the code of conduct <!-- https://github.com/micromark/.github/blob/main/code-of-conduct.md -->
+*   [ ] I searched issues and couldn’t find anything (or linked relevant results below) <!-- https://github.com/search?q=user%3Amicromark&type=Issues -->
```

Comment:
these links don't have an xdm equivalent yet, left them pointing to micromark for now.